### PR TITLE
Add previous versions and update to version 0.23.0

### PR DESCRIPTION
**Pull request by RaptaG**
Can you help me retrieve the hash for the `.tar.gz` @kolaente ?

### CODE REVIEW

**Comment by RaptaG** [comment]
> Build [123175 failed](https://buildbot.flathub.org/#/builders/6/builds/123175)

Because I have not added the hash yet

**Comment by kolaente** [comment]
Looks like the file failed to download?

**Comment by minionflo** [comment]
because somebody/something added a "v" before the version number in version 0.23.0

**Comment by minionflo** [comment]
For my Download the HASH is 6ac2492159fff5be0318409c95721a6d0b67e0a072874fe7592f8a86a257cbeb

File: io.vikunja.Vikunja.appdata.xml [hunk 1]
```diff
@@ -40,8 +40,11 @@
 	<url type="help">https://community.vikunja.io</url>
 	<releases>
 		<release version="0.21.0" date="2023-7-10" />
+		<release version="0.22.0" date="2023-12-19" />
+		<release version="0.22.1" date="2024-01-28" />
+		<release version="0.23.0" date="2024-02-01" />
 	</releases>
-	<developer id="io.vikunja">
+    <developer id="io.vikunja">
```
Comment (minionflo) [inline io.vikunja.Vikunja.appdata.xml]: i think the formater broke here

**Comment by RaptaG** [comment]
@Minionflo could you open a quick PR on the new-ver branch fixing the issues because I can't right now? I will merge it

**Comment by RaptaG** [comment]
Success! Now we wait for @kolaente to approve and this should be ready to go!
Thanks a lot for the major help in both Wayland support and updating the version!

**Comment by kolaente** [comment]
Thanks!

**Pull request merged**